src/main/java/billgates/usecases/insert_entry/InsertEntryRequestModel.java

```diff
@@ -12,21 +12,24 @@ public class InsertEntryRequestModel {
     private String from = "";
     private String to = "";
     private String location = "";
+    private final int SplitBillId;
+
 
     public InsertEntryRequestModel(ZonedDateTime date,
                                    double value,
                                    String currency,
                                    String description,
                                    String from,
                                    String to,
-                                   String location) {
+                                   String location, int SplitBillId) {
         this.date = date;
         this.value = value;
         this.currency = currency;
         this.description = description;
         this.from = from;
         this.to = to;
         this.location = location;
+        this.SplitBillId = SplitBillId;;
```

Comment:
Convention: variable name should be in the lower camel-case. For example, `SplitBillId` should be changed to `splitBillId`.